apps/ejabberd/src/mongoose_admin.erl

```diff
@@ -0,0 +1,268 @@
+-module(mongoose_admin).
+-author('bartlomiej.gorny@erlang-solutions.com').
+
+-export([start/0, stop/0,
+         start/2, stop/2,
+         register/3,
+         unregister/1,
+         registered_commands/0,
+         registered_users/1,
+         change_user_password/2,
+         listsessions/1,
+         kick_this_session/1,
+         get_recent_messages/3,
+         send_message/3
+        ]).
+
+-include("ejabberd.hrl").
+-include("jlib.hrl").
+
+start() ->
+    mongoose_commands:register(commands()).
+
+stop() ->
+    mongoose_commands:unregister(commands()).
+
+start(_, _) -> start().
+stop(_, _) -> stop().
+%%%
+%%% mongoose commands
+%%%
+
+
+commands() ->
+    [
+        [
+            {name, listmethods},
+            {category, list},
+            {desc, "List commands"},
```

Comment:
To be consistent with the code base the description should be binary as we are no longer use old strings to represent text in the code. This comment applies to all commands in this module.